### PR TITLE
Add the sentry-cli script and update appveyor to handle symbol uploading

### PR DESCRIPTION
**Pull request by RodrigoHolztrattner**
Automatically upload .sym files to Sentry for the backend

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ after_build:
   - copy "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.16.27012\x64\Microsoft.VC141.CRT\msvcp140.dll" "%SLFullDistributePath%\obs-studio-node\"
   - copy "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\ucrtbase.dll" "%SLFullDistributePath%\obs-studio-node\"
   - tar cvaf "%UnsignedArtifact%.tar.gz" -C "%SLFullDistributePath%" "obs-studio-node"
-
+
```
Comment (EddyGharbi) [inline appveyor.yml]: Would remove this extra space here that is adding more stuff to the diff

File: ci/run-sentry-cli.cmd [hunk 2]
```diff
@@ -0,0 +1,12 @@
+md syms
+cd "\projects\obs-studio-node\streamlabs-build\distribute\obs-studio-node\"
+for /f %%f in ('dir /b "\projects\obs-studio-node\streamlabs-build\distribute\obs-studio-node\*.pdb"') do "\projects\obs-studio-node\dump_syms.exe" %%f > "\projects\obs-studio-node\syms\%%~nf.sym"
+
+cd "\projects\obs-studio-node\streamlabs-build\distribute\obs-studio-node\obs-plugins\64bit\"
+for /f %%f in ('dir /b "\projects\obs-studio-node\streamlabs-build\distribute\obs-studio-node\obs-plugins\64bit\*.pdb"') do "\projects\obs-studio-node\dump_syms.exe" %%f > "\projects\obs-studio-node\syms\%%~nf.sym"
+
+cd "\projects\obs-studio-node\syms"
+
+"..\sentry-cli.exe" --auth-token "e035c0e6c9b74444a1765da04cf85c35f019520a897249ffa1ba17e9ffb3cd5b" upload-dif --org streamlabs-obs --project obs-server "..\syms"
```
Comment (EddyGharbi) [inline ci/run-sentry-cli.cmd]: Seems like we are uploading both of the client and server pdbs to obs-server here, could you split them and upload the correct pdbs for each project?

**Review by EddyGharbi — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 1 above)
Comment (RodrigoHolztrattner) [reply appveyor.yml]: Done!

File: ci/run-sentry-cli.cmd (hunk 2 above)
Comment (RodrigoHolztrattner) [reply ci/run-sentry-cli.cmd]: Done!

**Review by EddyGharbi — APPROVED** [review]
Looks good to me 👍 Merging

**Pull request merged**